api/streams.go

```diff
@@ -404,11 +406,13 @@ type Placement struct {
 
 // StreamSourceInfo shows information about an upstream stream source.
 type StreamSourceInfo struct {
-	Name     string          `json:"name"`
-	External *ExternalStream `json:"external,omitempty"`
-	Lag      uint64          `json:"lag"`
-	Active   time.Duration   `json:"active"`
-	Error    *ApiError       `json:"error,omitempty"`
+	Name                 string          `json:"name"`
+	External             *ExternalStream `json:"external,omitempty"`
+	Lag                  uint64          `json:"lag"`
+	Active               time.Duration   `json:"active"`
+	Error                *ApiError       `json:"error,omitempty"`
+	FilterSubject        string          `json:"filter_subject,omitempty"`
+	SubjectTransformDest string          `json:"subject_transform_dest,omitempty"`
```

Comment:
ship has probably sailed no but the `_dest` here is just redundant, its obvious from the context what it is.  The use of abbreviations like `dest` and `src` is a bad UX with no tangiable technical gain.

Reply:
Also need to update the json schema for this in `definitions.json` `stream_source_info`

Reply:
Yeah you are probably right on `_dest`, I have a tendency to err on the side of too verbose when picking names and in my head at least a transform always has a source and a destination (and here the source's field name was already set as filter_subject so didn't want to change that).

Fixed `stream_source_info` (also added filter_subject to it which had been missed)